src/v2/System/Analytics/Schema/Values.ts

```diff
@@ -146,6 +146,12 @@ export enum ActionType {
    */
   ClickedArtworkGroup = "clickedArtworkGroup",
   ClickedBuyViewingGroup = "clickedBuyViewingRoom",
+
+  /**
+   * Artwork Submission
+   */
+  SubmitAnotherArtwork = "submitAnotherArtwork",
+  ClickedFAQ = "clickedFAQ",
```

Comment:
**question:** I was expecting to also see the `clickedArtworkGroup` event represented in this PR. Is that no longer needed?

Reply:
Ah! I read the PR description after sending in my review... 🤦‍♂️